scripts/extensions/auto-tagging-widget/src/data-transformations.ts

```diff
@@ -63,7 +63,7 @@ export function getExistingTags(article: IArticle): IServerResponse {
                         name,
                         description,
                         qcode,
-                        source,
+                        source: source ?? '',
```

Comment:
It's not a good practice to use fake values like here. Empty string is not a valid source. It would be much better to simply make it optional in other interfaces in this extension.